openmc/mgxs/library.py

```diff
@@ -1410,6 +1420,8 @@ def check_library_for_openmc_mgxs(self):
           fixed source problem could be the target.
         - Fission and kappa-fission are not required as they are only
           needed to support tallies the user may wish to request.
+        - Scattering multiplicity should have been tallied, either using a
+        multiplicity or scatter and nu-scatter matrix tally.
```

Comment:
While folks using OpenMC CE to compute MGXS for OpenMC MG *should* be using multiplicity or nu-scatter data (otherwise they are losing some of the accuracy from the CE run that could be had relatively cheaply), people who are creating an MG model with MGXS sources from other sources of information (especially benchmark cases like C5G7) will probably not have scatter + nu-scatter or scatter+multiplicity information and they probably will not care since they just want to use the same data as the reference.

Now, since there may be a use-case where one would want to see the impact of including scattering multiplicity data or not, or they would want to replicate a benchmark model exactly as it was done in the benchmark, I originally chose not to add that check in this function. I'm ok with a warning, however (i.e., remove the error_flag = True part). Any thoughts on the matter as someone who just had to go through toe learning process of using the code?

Reply:
Agreed. OpenMC CE users should definitely think about including scattering multiplicity. I think it's worth around 200 pcm for a PWR, and benchmark users may not have it explicitly stated.
A warning is good for the first crowd.